global/global.go

```diff
@@ -0,0 +1,57 @@
+package global
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// appName is the command line name of the Arduino CLI executable on the user system (users may change it)
+var appName = filepath.Base(os.Args[0])
+
+var (
+	application= "arduino-cli"
```

Comment:
uhm this file doesn't look go fmt-ed:
https://travis-ci.org/arduino/arduino-cli/builds/541797053#L499

I see also that there are some other errors from the linter:
https://travis-ci.org/arduino/arduino-cli/builds/541797053#L502

Reply:
sorry again, solved in 4d432faeea68f7b85335abf2560d3235d0106dfb